libimage/layer_tree.go

```diff
@@ -89,21 +89,19 @@ func (l *layerNode) repoTags() ([]string, error) {
 	return orderedTags, nil
 }
 
-// layerTree extracts a layerTree from the layers in the local storage and
-// relates them to the specified images.
-func (r *Runtime) layerTree(ctx context.Context, images []*Image) (*layerTree, error) {
-	layers, err := r.store.Layers()
+// newFreshLayerTree extracts a layerTree from consistent layers and images in the local storage
+// and relates them to the all images.
```

Comment:
(I don’t know what the “and relates them to” part means, I think that can be dropped.)